vios/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+LABEL org.opencontainers.image.authors="xtothj@gmail.com"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -qy \
+   && apt-get upgrade -qy \
+   && apt-get install -y \
```

Comment:
we also use `--no-install-recommends` for apt to make the installation image smaller

Reply:
done